lib/ui/drawing_space/tool_options.dart

```diff
@@ -1,20 +1,59 @@
 import 'package:flutter/material.dart';
-import 'package:paintroid/ui/drawing_space/bottom_brush_tool_options.dart';
-import 'package:paintroid/ui/drawing_space/top_brush_tool_options.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:paintroid/tool/src/toolbox/toolbox_state_provider.dart';
+import 'package:paintroid/ui/drawing_space/stroke_cap_tool_option.dart';
+import 'package:paintroid/ui/drawing_space/tool_options_provider.dart';
+import 'package:paintroid/ui/drawing_space/stroke_width_tool_option.dart';
 
-class ToolOptions extends StatelessWidget {
+import 'package:paintroid/tool/src/tool_types.dart';
+
+class ToolOptions extends ConsumerWidget {
   const ToolOptions({super.key});
+  final animationDuration = const Duration(milliseconds: 300);
+  final maxOpacity = 1.0;
+  final minOpacity = 0.0;
 
   @override
-  Widget build(BuildContext context) {
-    return const Padding(
-      padding: EdgeInsets.all(8),
-      child: Column(
-        children: [
-          TopBrushToolOptions(),
-          Spacer(),
-          BottomBrushToolOptions(),
-        ],
+  Widget build(BuildContext context, WidgetRef ref) {
+    var options = getToolOptions(context, ref);
+    return  Padding(
+      padding: const EdgeInsets.all(8),
+      child: options,
+    );
+  }
+
+  Column getToolOptions(BuildContext context, WidgetRef ref) {
```

Comment:
Please make a widget out of these functions. https://medium.com/@s4.ali/flutter-code-review-dos-and-don-ts-and-best-practices-1-5d003035953e